### PR TITLE
[issue-f2] solution to issue in f2

### PR DESCRIPTION
**Pull request by jfjobidon**
here is my solution to issue

### CODE REVIEW

File: feature2/additional-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Here is the solution to issue in feature2
```
Comment (jfjobidon) [inline feature2/additional-file.txt:1]: good job

Comment (jfjobidon2) [reply feature2/additional-file.txt:1]: bin correk

**Review by jfjobidon2 — APPROVED** [review]
review accepted

**Comment by jfjobidon2** [comment]
merging...

**Pull request merged**